webapp/channels/src/components/admin_console/license_settings/enterprise_edition/enterprise_edition.scss

```diff
@@ -170,4 +178,45 @@
             }
         }
     }
+
+    // Entry SKU specific styles
+    .have-license-section {
+        margin: 24px 0;
+
+        .have-license-title {
+            margin: 0 0 4px 0;
+            color: #3f4350;
+            font-size: 16px;
+            font-weight: 600;
+            line-height: 24px;
+        }
+
+        .have-license-description {
+            margin: 0 0 10px 0;
+            color: rgba(63, 67, 80, 0.72);
+            font-size: 14px;
+            font-weight: 400;
+            line-height: 20px;
+        }
+
+        .upload-license-button-container {
+            .upload-license-btn {
+                display: inline-flex;
+                align-items: center;
+                padding: 8px 16px;
+                font-size: 14px;
+                font-weight: 600;
+                gap: 8px;
+
+                i {
+                    font-size: 16px;
+                }
+
+                &:disabled {
+                    cursor: not-allowed;
+                    opacity: 0.5;
+                }
+            }
+        }
```

Comment:
```suggestion
```

I don't think we need any of this since it's covered by using the btn btn-primary classes